autospec/license.py

```diff
@@ -126,12 +126,13 @@ def scan_for_licenses(srcdir):
                "notice",
                "copyrights",
                "about_bsd.txt"]
-    # look for files that start with copying or licen[cs]e (spelling errors)
-    # or end with licen[cs]e
+    # look for files that start with copying or licen[cs]e (but are
+    # not likely scripts) or end with licen[cs]e
     target_pat = re.compile(r"^((copying)|(licen[cs]e))|(licen[cs]e)$")
+    script_pat = re.compile(r"\.(sh|pl|py|rb)$")
```

Comment:
How do you feel about looking for shebang's to start the file in addition to/in place of this?

Reply:
I feel like it's a great idea. Trying now.

Looking at what files we have packaged as licences and would no longer be so, I can only find one: /usr/share/doc/autogen/agen5_test_license.test